iothub/service/src/Messaging/MessageFeedbackProcessorClient.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Subclient of <see cref="IotHubServiceClient"/> for handling cloud-to-device message feedback.
+    /// </summary>
+    /// <seealso href="https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messages-c2d#message-feedback"/>.
+    public class MessageFeedbackProcessorClient : IDisposable
```

Comment:
I don't follow how users are expected to use this class to get c2d message feedback. I was expecting to see a callback that executes each time feedback is received, but I don't see anything like that. All I see is the ability to open and close an AMQP connection

Reply:
Same comment for FileUploadNotificationProcessort

Reply:
I'd suggest you take a look at the [Java implementation](https://github.com/Azure/azure-iot-sdk-java/blob/main/service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/messaging/MessageFeedbackProcessorClient.java) of this class and the [associated sample](https://github.com/Azure/azure-iot-sdk-java/blob/main/service/iot-service-samples/message-feedback-processor-client-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/MessageFeedbackProcessorClientSample.java) to understand what this client is supposed to do.

Reply:
@timtay-microsoft added the suggested changes.